app/Http/Controllers/MessageController.php

```diff
@@ -29,7 +29,7 @@ public function index($userId)
         $usersTo = User::whereIn('id', $usersToIds)->get();
 
         return $this->setStatusCode(200)->respond(
-            $messages->makeHidden('created_at', 'updated_at'),
+            $messages->makeHidden(['created_at', 'updated_at']),
```

Comment:
@Pozharskyi probably fronrend will need those value to print when message was send and when it was changed